src/function/table/system_functions.cpp

```diff
@@ -23,6 +23,9 @@ void BuiltinFunctions::RegisterSQLiteFunctions() {
 	DuckDBDatabasesFun::RegisterFunction(*this);
 	DuckDBFunctionsFun::RegisterFunction(*this);
 	DuckDBKeywordsFun::RegisterFunction(*this);
+	TestLoggingFun::RegisterFunction(*this);
```

Comment:
maybe this could be a test-only extension like the `loadable_extension_test` or so